### ISSUE TITLE
Cannot see different green highlight in lux theme

### ISSUE THREAD

**Issue by Aspire1Inspire2** [liviuschera/noctis#41]
![image](https://user-images.githubusercontent.com/12798470/75714220-4c6a2d80-5c99-11ea-9371-ee253ce5ad0b.png)

I think the green color contrast should be increased a little bit?

**Comment by liviuschera** [comment]
Will you please describe how can I reproduce this issue?

**Comment by Aspire1Inspire2** [comment]
1. In VS Code, apply the Lux theme from noctis.
2. Use your preference diff tool or git tool etc. to compare two files/commits, etc.
3. The VS code will show difference. Red is removed. Green is added.
![image](https://user-images.githubusercontent.com/12798470/75995798-f086eb00-5eca-11ea-9590-37bef9ec18fb.png)
**_Figure "remov" should be marked as darker red. "add" should be marked as darger green._**

4. The darker red is the characters that are removed. Correspondingly, the darker green is the characters that are being added.


In the current Lux theme the contrast of light green and darker green is hard to tell.

**Comment by liviuschera** [comment]
Thanks mate, brilliant walk-through!

It should be fixed now. Let me know if it's what you're after.

Cheers!
